tests/tests_selinux_modules.yml

```diff
@@ -19,42 +19,50 @@
       block:
         - include_role:
             name: linux-system-roles.selinux
+      rescue:
+      - name: Cleanup modules
```

Comment:
indentation - the `-` should be indented two more spaces - please check the Travis CI results